README.md

```diff
@@ -213,16 +215,43 @@ Example:
    Hello, world!
 ```
 
+#### Code examples
+
+Use `code-block:: rust` to show Rust content with correct formatting.
+
+Use `code-block:: toml` to show TOML content with correct formatting.
+
+Use `code-block:: json` to show JSON content with correct formatting.
+
+Use `code-block:: jsx` to show JSX content with correct formatting.
+
+A particular line number may be emphasized with :emphasize-lines:line_number option. Multiple lines are comma separated (e.g. :emphasize-lines:10,12).
+
+#### Code inline in text
+
+Use ``code`` to insert commands or output from a terminal screen into a line of text. For example:
```

Comment:
```suggestion
Use ```code``` to insert commands or output from a terminal screen into a line of text. For example:
```

Reply:
We should go through the full document and find these issues once we decided on where we expect people to read it 😊